starlite/route_map.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Callable, Collection, Dict, List, Tuple
```

Comment:
Please add a comment on the class explaining it's a proxy for rust bindings. Also add #pragma: no cover top level, because this file is basically a shim.